app/_locales/de/messages.json

```diff
@@ -691,7 +691,7 @@
     "message": "Datenschutzrichtlinie"
   },
   "privateKeyWarning": {
-    "message": "Warnung: Niemals jemanden deinen Private Key mitteilen. Jeder der im Besitz deines Private Keys ist, kann jegliches Guthaben deines Accounts stehlen."
+    "message": "Warnung: Niemals jemandem deinen Private Key mitteilen. Jeder der im Besitz deines Private Keys ist, kann jegliches Guthaben deines Accounts stehlen."
```

Comment:
Google Translate shows both as `someone`; what's the reason for this change?  I don't know German, simply curious.

Reply:
The verb "mitteilen" requires the "Dativ", a special grammatical version of "jemand". As a proof, the German dictionary Duden shows the usage of mitteilen with "jemandeM" rather than "jemanden": 
https://www.duden.de/rechtschreibung/mitteilen
Example from that page: > "jemandem etwas brieflich, telefonisch, per E-Mail, schonend, vertraulich mitteilen"